team-telnyx/webrtc
# chore(WEBRTC-322): add integration tests to StoryBook components

- add integration tests to StoryBook components
- add cypress tools
- add data-testid tags
- configure cypress to run webrtc projects
- add storybook env environment

## 📝 To Do


## ✋ Manual testing

1. Navigate into `packages/js/examples/react-audio-widgets`
2. Create a `.env` file and put your credentials and a destination number.
3. Run `npm i && npm start`
4. Open a new terminal and run `npm run cypress:open`
5. Run the test on Firefox, Edge and Chrome.
6. You should get all green

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

| Description | Screenshot |
| ----------- | ---------- |
| Cypress tool |![image](https://user-images.githubusercontent.com/16343871/105848706-27239380-5fbe-11eb-9dec-ebe64bddd8a4.png)|
| test results |![image](https://user-images.githubusercontent.com/16343871/105848814-4b7f7000-5fbe-11eb-86d2-ac51c35171a6.png)|
